<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134,237,231);
</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>15</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;1、向传感器发送一帧时间信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>1、向传感器发送一帧时间信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>46</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>2、发送清除传感器历史数据指令</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>77</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>3、发送传感器数据清零测试指令</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>108</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>4、向上层设备发送分站本机参数信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>139</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>5、向上层设备发送分站关联传感器参数信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
